docs/src/modules/components/DemoToolbar.js

```diff
@@ -585,7 +585,7 @@ export default function DemoToolbar(props) {
             </ToggleButtonGroup>
           </Box>
         </Fade>
-        <Box sx={{ ml: 'auto', mr: 0.5 }}>
```

Comment:
@oliviertassinari just an FYI that this `mr: 0.5` was intentional! 😬 It was there to make the icon button container have the same distance from the container's edge as the language chips.

Reply:
fb6573e75e2de888322e90f925e3ab49a9df0878 should normally render exactly the same.

Reply:
Oh, sweet ⎯ that's cleaner 🤙